pkg/manifests/manifests.go

```diff
@@ -0,0 +1,536 @@
+package manifests
+
+import (
+	"encoding/json"
+	"os"
+
+	// "sort"
```

Comment:
It would be nice, since otherwise we don't have anything to prevent people from adding multiple image instances that are targeted at exactly the same platform to a list, but manually querying and then adding the specific instances of a tagged list like k8s.gcr.io/pause:3.1, where the list knows the individual instances are for different architectures, but all of the individual instances' configuration blobs list the same arch/OS information, confuses this logic.  I'll drop it out.